en/docs/_installations/alternatives.md

```diff
@@ -11,6 +11,16 @@ recommended to install Yarn via our packages instead.
 
 #### Install via npm
 
+> Note: Installation via npm is generally not recommended. Installing via npm is
+significantly slower than all of our other installation methods. npm packages
+are not signed and npm does not perform any integrity checks, which is a
+security risk when installing system-wide apps. Additionally, in an environment
+with a system-wide package manager such as most Linux distributions, apps
+installed via npm are not updated when you update the rest of your system.
```

Comment:
Hash verification is not the same. It doesn't verify **who** uploaded the
package or that it's actually a legitimate release, it's just a hash of the
contents. Additionally, npm uses SHA1, which is not considered secure.